src/App.tsx

```diff
@@ -1,61 +1,39 @@
 import './App.scss';
+import { AddTodoForm } from './components/AddTodoForm/AddTodoForm';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+
+function getUserByID(userId: number) {
+  return usersFromServer.find(user => user.id === userId);
```

Comment:
The `getUserByID` function assumes that a user will always be found. Consider handling the case where `find` returns `undefined` to avoid potential runtime errors.